tests/Feature/UsersTest.php

```diff
@@ -290,7 +290,8 @@ public function testDisableToken(): void
 		$oldToken = $users_test->reset_token()->offsetGet('token');
 		assertNotEquals('', $oldToken);
 
-		$newToken = $users_test->disable_token()->offsetGet('token');
+		$users_test->unset_token();
+		$newToken = $users_test->get_user()->offsetGet('token');
 		assertEquals('', $newToken);
```

Comment:
I am surprised. Shouldn't be that `null`?

Reply:
I don't know. But if it does I would prefer the following approach to have a stricter comparison.

```php
$userResponse = $users_test->get_user();
$userReponse->assertJson([
	'token' => null,
]);
```